receiver/filelogreceiver/README.md

```diff
@@ -18,6 +18,7 @@ Tails and parses logs from files.
 | `multiline`                  |                  | A `multiline` configuration block. See below for more details                                                      |
 | `force_flush_period`         | `500ms`          | Time since last read of data from file, after which currently buffered log should be send to pipeline. Takes `time.Duration` (e.g. `10s`, `1m`, or `500ms`) as value. Zero means waiting for new data forever |
 | `encoding`                   | `utf-8`          | The encoding of the file being read. See the list of supported encodings below for available options               |
+| `preserve_whitespace`        | `false`          | Whether to preserve leading and trailing whitespace. |
```

Comment:
Would you please also update the udp and tcp receiver readme's?

Reply:
updated readme.